src/tests/functional/inference_engine/snippets/src/pass/insert_load_store.cpp

```diff
@@ -0,0 +1,74 @@
+// Copyright (C) 2022 Intel Corporation
```

Comment:
Lets separate this tests to separated target because we try to test separated lib

Reply:
Ok, let's do it, but after all other fixes are applied, since the new binary won't be compiled and run in CI (at least until this pr is merged).

Reply:
The PR with connected changes. I mean use the product config

Reply:
Done, product config PR: https://github.com/intel-innersource/frameworks.ai.openvino.ci.product-configs/pull/1026